src/stylus/gene-label.styl

```diff
@@ -1,7 +0,0 @@
-.geniblocks.allele
```

Comment:
I couldn't figure out where this styling is used, and it was throwing off my drop downs, so I removed it. Do you know where I could see it used to maybe restrict this styling?